frontend/src/components/worksheets/items/TextEditorItem/TextEditorItem.js

```diff
@@ -93,7 +93,9 @@ class TextEditorItem extends React.Component<{
             type: 'POST',
             success: (data, status, jqXHR) => {
                 const moveIndex = true ? mode === 'create' : false;
-                const param = { moveIndex };
+                // No need to refresh the search results when only updating markdown items
+                const searchRefresh = false;
```

Comment:
Instead of making the prop `searchRefresh`, can we rename it to `skipSearchRefresh`? This way, it will be clearer what the prop does, and we only set it when we want to do something special (which is skipping the search refresh).

Reply:
Sure. Updated